rllib/algorithms/bc/bc.py

```diff
@@ -123,6 +123,11 @@ def validate(self) -> None:
         if self.beta != 0.0:
             raise ValueError("For behavioral cloning, `beta` parameter must be 0.0!")
 
+    @property
+    def _model_auto_keys(self):
```

Comment:
Do we even need this override, then?

Reply:
Another good one. You are right - this should automatical use the auto configs from the super.